modules/afsocket/socket-options-inet.c

```diff
@@ -231,9 +224,13 @@ socket_options_inet_new_instance(void)
   socket_options_init_instance(&self->super);
   self->super.setup_socket = socket_options_inet_setup_socket;
   self->super.so_keepalive = TRUE;
-#if defined(TCP_KEEPTIME) && defined(TCP_KEEPIDLE) && defined(TCP_KEEPCNT)
+#ifdef TCP_KEEPIDLE
```

Comment:
I would remove these ifdef statements, the member variables are always present, setting them to a value which is then ignored should make no harm.

Reply:
@bazsi : okay, that should be good.